src/snowflake/connector/result_set.py

```diff
@@ -118,10 +118,7 @@ def _report_metrics(self) -> None:
                 metrics.get(DownloadMetrics.parse.value),
             )
 
-    def _fetch_arrow_batches(
-        self,
-    ) -> Iterator[Table]:
-        """Fetches a all the results as Arrow Tables, chunked by Snowflake back-end."""
+    def __can_create_arrow_iter(self) -> None:
```

Comment:
I did this so that this function is private, because I don't expect it be used outside of this class. I referred to this: https://www.geeksforgeeks.org/private-methods-in-python/